alpaca/hypothesis_test.go

```diff
@@ -0,0 +1,259 @@
+package alpaca
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"encoding/base64"1
```

Comment:
Go build will fail because of the trailing 1 here. Because of this I'm guessing you haven't even built this, so to be honest I'm a bit sceptical about this PR as a whole.